src/components/AdminRegisterPage/index.jsx

```diff
@@ -66,12 +68,11 @@ const AdminRegisterPage = ({ authentication, match }) => {
 };
 
 AdminRegisterPage.propTypes = {
-  authentication: PropTypes.shape().isRequired,
```

Comment:
__worth a quick check:__ not 100% on this but I thought there was some eslint rule or something else maybe that suggested using shape({}) instead of just shape() but don't take my word for it

Reply:
Yep, that is correct. I'm removing this entirely so it's moot here, though. :)